docs/pages/admin-guides/management/admin/troubleshooting.mdx

```diff
@@ -184,7 +184,8 @@ through the [Teleport support portal](https://support.goteleport.com).
 <TabItem scope={["oss"]} label="Teleport Community Edition">
 If you need help, please ask on our [community forum](https://github.com/gravitational/teleport/discussions). You can also open an [issue on GitHub](https://github.com/gravitational/teleport/issues).
 
-For more information about custom features, or to try the [self-hosted Enterprise edition](../../deploy-a-cluster/deploy-a-cluster.mdx) of Teleport, reach out to us at [sales](https://goteleport.com/signup/enterprise/).
+For more information about Enterprise features reach out to us at [sales](https://goteleport.com/signup/enterprise/).
+You can also sign up for a [free trial](https://goteleport.com/signup) of Teleport Enterprise (Cloud).
```

Comment:
```suggestion
You can also sign up for a [free trial](https://goteleport.com/signup) of Teleport Enterprise.
```

I think in this case, it's okay to leave the deployment mode unspecified, since a user could still enjoy most self-hosted Enterprise features using the cloud-hosted deployment mode.